integration-cli/docker_cli_daemon_plugins_test.go

```diff
@@ -246,7 +246,7 @@ func (s *DockerDaemonSuite) TestGraphdriverPlugin(c *check.C) {
 	c.Assert(err, checker.IsNil, check.Commentf(out))
 
 	// restart the daemon with the plugin set as the storage driver
-	s.d.Restart(c, "-s", plugin, "--storage-opt", "overlay2.override_kernel_check=1")
+	s.d.Restart(c, "--experimental", "-s", plugin, "--storage-opt", "overlay2.override_kernel_check=1")
```

Comment:
Lets make the testRequires an `ExperimentalDaemon` instead. Easier to switch to stable, when its supported.